frontend/src/components/UserUpdateRoles.vue

```diff
@@ -218,34 +389,54 @@ export default {
           this.userId,
           this.selectedClient.id,
           rolesToAdd
-        )
+        ),
       ])
         .then(() => {
           this.getUserClientRoles();
           this.$store.commit("alert/setAlert", {
             message: "Roles updated successfully",
-            type: "success"
+            type: "success",
           });
-          window.scrollTo(0, 0);
           //Update list of roles from UserDetails module
-          this.$root.$refs.UserDetails.loadUserRoles();
+          this.close();
+          this.loadUserRoles();
         })
         .catch((error) => {
           this.$store.commit("alert/setAlert", {
             message: "Error updating roles: " + error,
-            type: "error"
+            type: "error",
           });
-          window.scrollTo(0, 0);
-        });
-    }
-  }
+        })
+        .finally(() => {
+          this.$root.$refs.UserMailboxAuthorizations.getMailboxClients();
+          window.scrollTo(0, 0);})    
+    },
+  },
 };
 </script>
 
 <style>
-.right-gutters .col {
-  padding: 10px 12px 10px 0;
+.updateRolesDialog {
+    margin: 24px;
+    /* overflow-y: auto;
```

Comment:
These commented out CSS properties are why the FMDB and WEBCAPS clients are spilling over as per my review comment. I'll make the change directly.